go/exorelay/exorelay_test.go

```diff
@@ -17,16 +17,18 @@ import (
 func FeatureContext(s *godog.Suite) {
 	var exoInstance *ExoRelay
 	exocom := exocomMock.New()
-	go exocom.Listen(4100)
+	go func() {
+		err := exocom.Listen(4100)
+		if err != nil {
+			fmt.Printf("ERROR: cannot start exocomMock: %s", err)
```

Comment:
Thoughts on `log.Fatal` here? I don't think we want to continue running if this fails